index.html

```diff
@@ -755,6 +744,14 @@ <h2>
             </li>
           </ul>
         </dd>
+        <dt>
+          <dfn>Patronizing</dfn>
+        </dt>
+        <dd>
+          <p>
+            Language or behaviour that may appear kind or helpful but betrays a feeling of superiority or condescension.
+          </p>
+        </dd>
```

Comment:
It seems like an odd choice to add a term definition for something that it used only once, and is explained in more depth in the place where it is used.

If keeping this, I would suggest changing "betrays" to "conveys" because I don't think we can constrain what people feel, even if they're feeling things we'd rather they didn't, but we can set constraints on how people behave.